src/version/get-versions.ts

```diff
@@ -89,9 +89,10 @@ export function getOldestVersions(
   ).pipe(
     map(result => {
       if (result.repository.packages.edges.length < 1) {
-        throwError(
+        console.log(
```

Comment:
is this added for testing or are we going to swallow error?

Reply:
The throwError was not working as expected and breaking the action run. Hence we are giving the warning message and exiting the action without causing it to fail.